pymongo/message.py

```diff
@@ -1273,9 +1273,9 @@ def unpack_response(self, cursor_id=None,
         return bson._decode_all_selective(
             self.payload_document, codec_options, user_fields)
 
-    def command_response(self):
+    def command_response(self, codec_options=_UNICODE_REPLACE_CODEC_OPTIONS):
```

Comment:
Since we only call this method in one place, let's make codec_options a required argument to help prevent this kind of bug in the future.

Reply:
I also changed `_OpReply.command_response()` for uniformity.